Dockerfile

```diff
@@ -14,6 +14,11 @@ RUN \
   # Clean apt cache
   apt-get clean && \
   apt-get remove -y git && \
-  apt-get -y autoremove
+  apt-get -y autoremove && \
+  npm install pm2@3.2.2 -g
```

Comment:
I suggest to move this up in the RUN script, just after the other `npm install` command in line 11 (most probably grunt-cli installation will dispaear, but, by the moment keep it and add the new install just after it).